iris-mpc-cpu/src/execution/hawk_main/rot.rs

```diff
@@ -0,0 +1,77 @@
+use std::ops::Deref;
+
+use iris_mpc_common::ROTATIONS;
+use itertools::Itertools;
+
+/// VecRots are lists of things for each rotation.
+pub struct VecRots<R> {
+    pub rotations: Vec<R>,
```

Comment:
does this need to be pub since we have the Deref below? I think this could maybe confuse/break since we essentially assume a given size but it is modifiable from the outside. Also maybe `[R; ROTATIONS]` is an option?

Reply:
Right, I removed the `pub`.

Overall, the goal of this module is to clarify the complicated code that use it by naming things and add some type safety, so I went for Vec before fixed arrays, impl iterators, lifetimes, etc.